### ISSUE TITLE
Papercut: alt-text on fullscreen images are annoying

### ISSUE THREAD

**Issue by anoadragon453** [vector-im/element-web#18037]
So there I was, sitting in the TWiM room enjoying some late-night chatter about hacking doorbell systems - when I came across an image of a pcb. I squinted, attempting to stare at the tiny inscriptions upon the green metal, when I realised. "how foolish of me, Element has image zoom. We live in fruitful times."

I clicked the image to make it fullscreen and zoomed in, confident in my complete lack of understanding of what the circuit was actually doing. But as my eyes scanned across, trying to make sense of the hieroglyphs, a box appeared on top of the image out of nowhere, displaying: **image.png**.

![image](https://user-images.githubusercontent.com/1342360/126017296-5932a4bd-9d7c-41a1-93ee-9209682ad822.png)

My attention was lost, and instead I suddenly had the urge to make a bug report rather than pretending to understand circuit boards :(

TL;DR it's annoying that the image filename appears as alt text when you hover over it with your mouse, especially when this information is already available elsewhere.

#### Steps to reproduce

- Open an image from a room's timeline.
- Hover over it with your mouse.
- Alt-text appears.

#### Version information

- **Platform**: Desktop
- **OS**: Linux
- **Version**: Nightly 2021071601
